<launch>
  <arg name="gripper" default="r_gripper" />
  <arg name="fingertip" default="l_fingertip" />

  <node name="pfs_force_plot"
        pkg="rqt_plot"
        type="rqt_plot"
        args="/pfs/$(arg gripper)/$(arg fingertip)/force[0]
              /pfs/$(arg gripper)/$(arg fingertip)/force[1]
              /pfs/$(arg gripper)/$(arg fingertip)/force[2]
              /pfs/$(arg gripper)/$(arg fingertip)/force[3]
              /pfs/$(arg gripper)/$(arg fingertip)/force[4]
              /pfs/$(arg gripper)/$(arg fingertip)/force[5]
              /pfs/$(arg gripper)/$(arg fingertip)/force[6]
              /pfs/$(arg gripper)/$(arg fingertip)/force[7]
              /pfs/$(arg gripper)/$(arg fingertip)/force[8]
              /pfs/$(arg gripper)/$(arg fingertip)/force[9]
              /pfs/$(arg gripper)/$(arg fingertip)/force[10]
              /pfs/$(arg gripper)/$(arg fingertip)/force[11]
              /pfs/$(arg gripper)/$(arg fingertip)/force[12]
              /pfs/$(arg gripper)/$(arg fingertip)/force[13]
              /pfs/$(arg gripper)/$(arg fingertip)/force[14]
              /pfs/$(arg gripper)/$(arg fingertip)/force[15]
              /pfs/$(arg gripper)/$(arg fingertip)/force[16]
              /pfs/$(arg gripper)/$(arg fingertip)/force[17]
              /pfs/$(arg gripper)/$(arg fingertip)/force[18]
              /pfs/$(arg gripper)/$(arg fingertip)/force[19]
              /pfs/$(arg gripper)/$(arg fingertip)/force[20]
              /pfs/$(arg gripper)/$(arg fingertip)/force[21]
              /pfs/$(arg gripper)/$(arg fingertip)/force[22]
              /pfs/$(arg gripper)/$(arg fingertip)/force[23]" />
</launch>
